website/project/views/contributor.py

```diff
@@ -665,7 +665,7 @@ def check_external_auth(user):
 
 @block_bing_preview
 @collect_auth
-@must_be_valid_project
+@must_be_valid_project(preprints_valid=True)
```

Comment:
Allows logged-in users to claim an unregistered account as their own (following link from a preprints email)
<img width="719" alt="screen shot 2018-12-02 at 3 06 09 pm" src="https://user-images.githubusercontent.com/9755598/49345374-850bf900-f649-11e8-8c43-9be4fd261733.png">